.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+  - "6"
+  - "8"
+cache:
+  directories:
+    - /home/travis/build/technopagan/sqip/vendor
```

Comment:
"vendor"? Just know this from PHP. Is this really needed?

Reply:
U are probably correct here, too. I will remove it and we will see :)